os/drivers/iotdev/iotbus_sig.c

```diff
@@ -110,14 +110,14 @@ static int iotbus_sig_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 
 		struct iotbus_int_info_s *data = (struct iotbus_int_info_s *)kmm_zalloc(sizeof(struct iotbus_int_info_s));
 		if (data == NULL) {
-			return IOTBUS_ERROR_OUT_OF_MEM;
+			return -EINVAL;
```

Comment:
ENOMEM

Reply:
@Taejun-Kwon 
I fixed it. Thanks a lot.